data/core.yaml

```
@@ -1971,6 +1971,8 @@ en:
         title: Continue drawing from start
       continue_from_end:
         title: Continue drawing from end
+      tag_as_entrance:
+        title: Mark the endpoint as a {presetName}
```

Ideally this would be rephrased to remove the indefinite article, any suggestions would be appreciated.

---

I tend to use this pattern in similar situations

```suggestion
        title: Mark the endpoint as »{presetName}«
```